src/main/java/org/assertj/core/error/ShouldHaveScale.java

```diff
@@ -0,0 +1,12 @@
+package org.assertj.core.error;
+
+public class ShouldHaveScale extends BasicErrorMessageFactory {
+
+  public static ErrorMessageFactory shouldHaveScale(int actual, int expected) {
+    return new ShouldHaveScale(actual, expected);
+  }
+
+  private ShouldHaveScale(int actual, int expected) {
+    super("%nexpecting actual to have a scale of: %s%n  but has a scale of: %s", expected, actual);
```

Comment:
let's pass actual so that we can display in the error message as in:
```
"%nexpecting actual %s to have a scale of:%n  %s%nbut had a scale of:%n  %s", actual, expected, actual.getScale()
```
with this error message (if I got it correctly) scales are aligned in the output